src/api/goods.ts

```diff
@@ -10,10 +10,13 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods
+      .sort((firstGood, secondGood) => firstGood.name
```

Comment:
```suggestion
    .then(goods => [...goods]
      .sort((firstGood, secondGood) => firstGood.name
```
Sort is a mutable method, so it's better to make a copy